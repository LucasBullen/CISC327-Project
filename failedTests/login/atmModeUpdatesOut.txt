1,7c1,7
< Select mode: atm or agent.
< Logged into agent mode.
< Logged out.
< Select mode: atm or agent.
< Logged into atm mode.
< Cannot access create feature in atm mode.
< Logged out.
---
> Select mode: atm or agent.
> Logged into agent mode.
> Logged out.
> Select mode: atm or agent.
> Logged into atm mode.
> Cannot access create feature in atm mode.
> Logged out.
