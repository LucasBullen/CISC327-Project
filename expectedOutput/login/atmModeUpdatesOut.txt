Select mode: atm or agent.
Logged into agent mode.
Logged out.
Select mode: atm or agent.
Logged into atm mode.
Cannot access create feature in atm mode.
Logged out.
